src/components/Comments/ConnectorWrapper.js

```diff
@@ -22,8 +22,8 @@ export const styles = StyleSheet.create({
     borderLeftWidth: 1,
     borderBottomWidth: 2,
     borderBottomLeftRadius: 21,
-    borderLeftColor: index === 0 ? colors.gray1 : 'transparent',
-    borderBottomColor: index === 0 ? colors.gray1 : 'transparent'
+    borderLeftColor: index === 0 ? COLORS.balance_gray : 'transparent',
+    borderBottomColor: index === 0 ? COLORS.balance_gray : 'transparent'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color references in the style properties have been updated to use the new `COLORS` object. Again, this aligns with the PR's objective. Ensure that `COLORS.balance_gray` exists and is the intended color.
```diff
-    borderLeftColor: index === 0 ? colors.gray1 : 'transparent',
-    borderBottomColor: index === 0 ? colors.gray1 : 'transparent'
+    borderLeftColor: index === 0 ? COLORS.balance_gray : 'transparent',
+    borderBottomColor: index === 0 ? COLORS.balance_gray : 'transparent'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->